python/test/test_variableslice.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+from dataset import *
+import numpy as np
+
+
+class TestVariableSlice(unittest.TestCase):
+
+    def setUp(self):
+        d = Dataset()
```

Comment:
Why are you using `Dataset` and `DatasetSlice` in a test of `VariableSlice`? Why not simply work with `Variable` and slice that? Like this, the tests is less self-contained.